core/architect_bot.py

```
@@ -184,12 +229,109 @@ async def handle_quick_docs(message, say, context):
+                        if summary_text:
+                            await client.chat_postMessage(channel=channel, thread_ts=thread_ts, text=summary_text)
+                    except Exception as e:
+                        logger.error(f"URL Thread summarization failed: {e}", exc_info=True)
+                        await client.chat_postMessage(channel=channel, thread_ts=thread_ts, text=f"❌ I ran into an issue trying to summarize the linked thread: {str(e)}")
+                else:
+                    # Current thread summarization
+                    if 'thread_ts' not in event:
+                        await say("I can only summarize conversations in a thread. Please mention me in a reply to the thread you want summarized.")
+                        return
+                    
+                    await say(f"Sure, I'll summarize this thread for you. One moment... 🧐")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)